corpus/accounts/forms.py

```diff
@@ -61,3 +62,12 @@ def clean_username(self):
         if username:
             username = username.lower()
         return username
+
+
+class CorpusPasswordResetForm(PasswordResetForm):
+    def __init__(self, *args, **kwargs):
+        super(CorpusPasswordResetForm, self).__init__()
+        for visible in self.visible_fields():
+            visible.field.widget.attrs[
+                "class"
+            ] = "mt-1 block w-full rounded-md border-base-800 text-black shadow-sm focus:border-primary focus:ring focus:ring-primary-200 focus:ring-opacity-50"  # noqa: E501
```

Comment:
Remind me to fix this in my Achievements page PR. I've made changes to the form fields in that PR, will need to update this form accordingly.

Reply:
Actually, where is this being used? I don't see any reference to this Form instance. Also, the initial forgot password page does not have any styling classes applied to the input fields:

![image](https://github.com/IEEE-NITK/corpus/assets/66667161/a10630ce-cef2-4fa1-8091-938c3d776b81)

Reply:
Yeah, I think I should remove that. When I do subclass it to add our styles, idk why but the form functionality doesn't work. For now, I'll remove it, will debug properly and figure out how to do this later.

Reply:
Can you create an issue in the backlog for this?